src/UI/CustomWidgets/DataPreparation/DataGridWidget.py

```diff
@@ -48,3 +86,8 @@ def _on_show_grid_button_clicked(self, _) -> None:
     def _on_widget_state_change(self) -> None:
         """Callback for parent widget ensemble."""
         self.grid_output.clear_output()
+
+    def _on_set_grid_button_clicked(self, _) -> None:
+        self.data_manager.set_range_grid(self._row_range.value[0], self._row_range.value[1],
+                                         self._col_range.value[0], self._col_range.value[1]
+                                         )
```

Comment:
Using ranges of columns and rows as parameters for `show_data_grid()`, you won't need this function at all.